l10n.toml

```diff
@@ -103,5 +103,69 @@ branches = [
 [env]
 
 [[paths]]
-  reference = "components/**/src/main/res/values/strings.xml"
-  l10n = "components/**/src/main/res/values-{android_locale}/strings.xml"
+  reference = "components/browser/engine-system/src/main/res/values/strings.xml"
```

Comment:
@Pike it looks like the automated PRs are undoing your wild-card change.

Reply:
Yeah, sorry, I didn't realize that trying to fix another problem updated these PRs each time.

Good news, I have a fix in hand (and it also updated the PRs on android-l10n). The round after next should be fine.

Reply:
Should we then, close this pr and wait for the new update?

Reply:
I'd leave this open as a trash can until https://github.com/mozilla-l10n/android-l10n/pull/244 is merged.

The code bits are done, but there's also a ton of new strings in the a-c -> android-l10n PR, that @Delphine needs to go through.

Reply:
👍

Reply:
I've been reviewing mozilla-l10n/android-l10n#244  as the commits came in.
If it's safe to merge now, I can definitely do that. Please just let me know so that I don't make anything explode! 😜

Reply:
Heads-up that I've merged mozilla-l10n/android-l10n#244

Reply:
Thanks!